examples/web-sdk-sample/packages/vue2/src/components/UploadFile.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div class='list row'>
+    <div class='col-md-12'>
+      <h4>Upload file to S3 using AWS SDK</h4>
+      <div class='flex-container'>
+        <input type='file' ref='fileInput' />
+        <button class='btn btn-sm btn-success' @click='uploadFile'>
+          Upload
+        </button>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { S3Client, PutObjectCommand } from '@aws-sdk/client-s3';
+import { fromCognitoIdentityPool } from '@aws-sdk/credential-providers';
+
+const REGION = 'your bucket region';
+
+const s3Client = new S3Client({
+  region: REGION,
+  credentials: fromCognitoIdentityPool({
+    clientConfig: { region: REGION },
+    identityPoolId: 'your identityPoolId with S3 PutObject permission'
+  })
+});
+
+export default {
+  name: 'upload_file',
+  methods: {
+    async uploadFile() {
+      const fileInput = this.$refs.fileInput;
+      if (fileInput.files.length > 0) {
+        const file = fileInput.files[0];
+        console.log(file);
+        const command = new PutObjectCommand({
+          Bucket: 'your S3 bucket',
+          Key: file.name,
+          Body: file
+        });
+        try {
+          const response = await s3Client.send(command);
+          console.log(response);
+          if (response.$metadata.httpStatusCode === 200) {
+            console.log('upload success');
```

Comment:
Display success in ui or alert success message after upload success?

Reply:
added uploading and upload result display